frontend/pages/media/podcast.js

```diff
@@ -41,14 +44,10 @@ const Podcast = () => {
   // input: podcasts data from google sheets
   // output: array of dictionaries containing podcasts data
   const fetchPodcastEpisodes = async () => {
-    const res = await axios.get(
-      "https://wit-database.herokuapp.com/podcast-episodes"
-    );
-    const unsorted = humps.camelizeKeys(res.data);
-    const sortedEpisodes = unsorted.reverse();
-    setContent(sortedEpisodes);
-    // setCurrentPosts(sortedEpisodes);
-    setSelectedPosts(sortedEpisodes);
+    const res = await getPodcastEpisodes();
+    const allEpisodes = humps.camelizeKeys(res);
+    setContent(allEpisodes);
+    setSelectedPosts(allEpisodes);
```

Comment:
Consider including some logic to ensure that data is being sorted in the correct order